barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -513,14 +515,14 @@ template <typename Curve> class ZeroMorphVerifier_ {
      * @param concatenation_groups_commitments
      * @return Commitment
      */
-    static Commitment compute_C_Z_x(std::vector<Commitment> f_commitments,
-                                    std::vector<Commitment> g_commitments,
+    static Commitment compute_C_Z_x(const std::vector<Commitment>& f_commitments,
+                                    const std::vector<Commitment>& g_commitments,
                                     std::vector<Commitment>& C_q_k,
                                     FF rho,
                                     FF batched_evaluation,
                                     FF x_challenge,
                                     std::vector<FF> u_challenge,
-                                    const std::vector<std::vector<Commitment>>& concatenation_groups_commitments = {})
```

Comment:
The new shape here is vector<RefVector<>>. This required some adapting in the tests